<?xml version="1.0"?>
<launch>
    <node pkg="rcvm_matrice" type="server.py" name="rcvm_server" respawn="true" respawn_delay="10">
    </node>
</launch>
